stellar/operators.js

```diff
@@ -0,0 +1,72 @@
+const { Contract, Address } = require('@stellar/stellar-sdk');
+const { Command, Option } = require('commander');
+const { getWallet, prepareTransaction, buildTransaction, sendTransaction, estimateCost } = require('./utils');
+const { loadConfig, printInfo } = require('../evm/utils');
+
+require('dotenv').config();
+
+async function processCommand(options, _, chain) {
+    const [wallet, server] = await getWallet(chain, options);
+
+    const contract = new Contract(options.address || chain.contracts?.axelar_operators?.address);
+
+    let operation;
+
+    const address = Address.fromString(options.args || wallet.publicKey());
+
+    switch (options.action) {
+        case 'is_operator':
+            operation = contract.call('is_operator', address.toScVal());
+            break;
+        case 'add_operator':
+            operation = contract.call('add_operator', address.toScVal());
+            break;
+        case 'remove_operator':
+            operation = contract.call('remove_operator', address.toScVal());
+            break;
+        default:
+            throw new Error(`Unknown action: ${options.action}`);
+    }
+
+    if (options.estimateCost) {
+        const tx = await buildTransaction(operation, server, wallet, chain.networkType, options);
+        await estimateCost(tx, server);
+        return;
+    }
+
+    const signedTx = await prepareTransaction(operation, server, wallet, chain.networkType, options);
+    const returnValue = await sendTransaction(signedTx, server);
+    printInfo('is_operator', returnValue);
+}
+
+if (require.main === module) {
+    const program = new Command();
+
+    program.name('operators').description('Operators contract management');
+
+    program.addOption(
```

Comment:
Instead of adding a separate method, I'd like to refactor and reuse the EVM `addBaseOptions` to be chain agnostic. It'll be a separate task:
https://axelarnetwork.atlassian.net/browse/AXE-3980